src/events/events.service.ts

```diff
@@ -284,20 +300,17 @@ export class EventsService {
       _sum: {
         points: true,
       },
+      _count: {
+        points: true,
+      },
```

Comment:
@rohanjadvani I've added changes to `/metrics` route, since the main bottleneck was also in EventsService